scripts/filter_multimappers.py

```diff
@@ -160,5 +168,5 @@ def main(sam_file: Path) -> None:
 
 
 if __name__ == "__main__":
-    args = parse_arguments().parse_args()
-    main(sam_file=args.infile)
+    args = parse_arguments().parse_args() #pragma:no cover
+    main(sam_file=args.infile) #pragma: no cover
```

Comment:
It is actually possible to test code in the `__name__ == "__main__" trip. See [here](https://github.com/zavolanlab/zarp-cli/blob/b5214a255498be99b9d44e7c45bf157246f50004/tests/test_cli.py#L23-L29) for example.

But I'm also fine with skipping it. But then maybe use the typical convention for inline comments (2 white spaces before and 1 after the `#`) and consistent white space, e.g.:

```python
args = parse_arguments().parse_args()  # pragma: no cover
main(sam_file=args.infile)  # pragma: no cover
```